ArmVirtPkg/PlatformCI/.azurepipelines/Ubuntu-GCC.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   - job: Platform_CI
     variables:
       package: 'ArmVirtPkg'
-      vm_image: 'ubuntu-22.04'
+      vm_image: 'ubuntu-24.04'

```

Comment:
Kind of orthogonal to this PR, but can we also use ubuntu-24.04-arm here?

Reply:
Since we're using public images, I can only use what is available here - [Available Images](https://github.com/actions/runner-images?tab=readme-ov-file#available-images). Technically, we're just using the Ubuntu VM to load the [fedora-40-test](https://github.com/tianocore/containers/pkgs/container/containers%2Ffedora-40-test) image which includes the [arm flavors of tools](https://github.com/tianocore/containers/tree/main/Fedora-40).